BezierKit/Library/PathComponent+Arc.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  PathComponent+Arc.swift
+//  BezierKit
+//
+//  Created by ky1vstar on 15.10.2020.
+//  Copyright © 2020 Holmes Futrell. All rights reserved.
+//
+
+import Foundation
+
+extension PathComponent {
+    public convenience init(
```

Comment:
let's make the public interface on `Path` with an interface `init(ellipseIn rect: CGRect)`. I'm trying to keep the API as similar to `CGPath` and SwiftUI's `Path` as much as possible which means it's desirable for the API to operate on `Path` objects instead of `PathComponent` objects where possible. For now I think that also means hiding the `numberOfCurves` parameter as well (we can expose it later on potentially, it's just harder to take public API functionality away than to add it).